battleship/Form1.cs

```diff
@@ -12,9 +12,26 @@ namespace battleship
 {
     public partial class Form1 : Form
     {
+        WMPLib.WindowsMediaPlayer musicPlayer = new WMPLib.WindowsMediaPlayer();
         public Form1()
         {
             InitializeComponent();
         }
+
+        private void Form1_Load(object sender, EventArgs e)
+        {
+            musicPlayer.URL = (@"StartMusic.mp3");
+            musicPlayer.settings.volume = 15;
+            musicPlayer.controls.play();
+        }
+
+        private void beginButton_Click(object sender, EventArgs e)
+        {
+            startScreen.Enabled = false;
+            startScreen.Visible = false;
+            beginButton.Enabled = false;
+            beginButton.Visible = false;
+            musicPlayer.controls.stop();
```

Comment:
Just when the player clicks the begin button, to have an audio transition to the beginning of
 the game.